### ISSUE TITLE
browser-sync cann't open index.html

### ISSUE THREAD

**Issue by Spring8848** [BrowserSync/browser-sync#1477]
### Issue details
after installing npm and browsersync,i run the following command:
rowser-sync start --server --files "css/*.css,*.html"

![image](https://user-images.githubusercontent.com/20620007/34479511-520ecd84-efe2-11e7-86b3-77c5709f00cc.png)

a popup window does not show and render the default page--index.html,but listing a direcotry.
![image](https://user-images.githubusercontent.com/20620007/34479525-5e1cd1c0-efe2-11e7-9604-3756778722ff.png)

when i click the index.html,the browser shows 
![image](https://user-images.githubusercontent.com/20620007/34479867-02671d66-efe4-11e7-93df-d9b5ab62c508.png)

### Please specify which version of Browsersync, node and npm you're running

- Browsersync [  2.23.2  ]
- Node        [  v8.9.3  ]
- Npm         [   5.5.1 ]

### Affected platforms

- [ ] windows

**Comment by Kanchil** [comment]
same question

**Comment by apieceofbart** [comment]
I confirm on Ubuntu, Node v6.11.0, npm 3.10.10

**Comment by apieceofbart** [comment]
Another workaround is to add `--serveStatic "./"

**Comment by Spring8848** [comment]
@apieceofbart  that's work ,you are great !

**Comment by Glaived** [comment]
same issue with the following command:

```
D:\HTML browser-sync
λ browser-sync start -s -f "**/*" --no-open --no-notify
```

### Please specify which version of Browsersync, node and npm you're running

Browsersync [ 2.23.2 ]
Node [ v8.9.3 ]
Npm [ 5.6.0 ]

### Affected platforms
Windows

### Another solution

You can downgrade your browser-sync to v2.23.1 or 2.22.0. e.g. : `npm install -g browser-sync@2.23.1`

**Comment by Rick-Kirkham** [comment]
I'm still getting this on 2.23.3

**Comment by jokkopp** [comment]
same issue on 3.10.10

**Comment by shakyShane** [comment]
@Rick-Kirkham can you provide info about your dev environment?

**Comment by jokkopp** [comment]
windows 10 pro
node [ v6.10.2 ]
npm [ 3.10.10 ]
browser-sync [ 2.23.2 ]

windows 10 pro

**Comment by Rick-Kirkham** [comment]
@shakyShane 
node: 6.10.2
npm: 3.10.10
Windows 10

**Comment by shakyShane** [comment]
@Spring8848 @Rick-Kirkham I cannot re-create this issue any longer with 2.23.3 - could you provide any more information, or a screencast? Have you cleared any browser caches/service workers?

**Comment by shakyShane** [comment]
@Glaived could you try version 2.23.3 - your example works for me

**Comment by shakyShane** [comment]
@raphaelcockx this is due to the way browser-sync receives strings from CLI input on windows.

basically if you provide this: 

```sh
browser-sync start --server 'test'
```

internally Browsersync gets `server: "'test'"` (notice the single quotes remain) - not sure if there's a better way to handle this on windows, but for now always provide double-quoted arguments!

**Comment by Rick-Kirkham** [comment]
@shakyShane 
I have cleared the browser caches. I have rebooted, which should clear out all service workers. 
I've also tried manually deleting the `browser-sync*` folders in `node_modules` and in `...AppData\Roaming\npm-cache`.

I can make the issue come and go by switching between 2.23.3 and any version before 2.22.0 with 
`npm install browser-sync@n.nn.nn`
It happens on IE, Edge, Chrome, and Firefox.
The command in package.json is: `browser-sync start --config bsconfig.json`
My bsconfig is:
```
{
    "ui": {
        "port": 3000
    },
    "server": {
        "routes": {
            "/node_modules": "node_modules"
        }
    },
    "https": {
        "key": "./certs/server.key",
        "cert": "./certs/server.crt"
    },
    "watch": true,
    "files": "*.*"
}
```
I'm using HTTPS, but I've tried with HTTP and get the same results. 
Let me know if there's anything else I can provide.

**Comment by shakyShane** [comment]
@Rick-Kirkham 

with that server configuration

```
"server": {
        "routes": {
            "/node_modules": "node_modules"
        }
    }
```

what are you expecting to happen? serve the current directory + node_modules? (just trying to narrow in on the use-case)

**Comment by shakyShane** [comment]
@Rick-Kirkham I ask because that configuration wouldn't work on any version (unless I'm missing something)

when providing an Object for the `server` property, you should always specify a base directory also, eg:

```json
{
  "server": {
    "baseDir": "./app",
    "routes": {
      "/node_modules": "node_modules"
    }
  }
}
```

**Comment by Rick-Kirkham** [comment]
@shakyShane 
Fixed now. It was the missing baseDir property, which in my case is "./".  
Still puzzled, though, that it worked on versions below 2.22.0.

**Comment by shakyShane** [comment]
Closing now as OP reports this as fixed! 

sorry for any inconvenience caused :)

@Rick-Kirkham I'm going to push a separate fix for your case (browser-sync@2.23.5)
